sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -819,21 +835,29 @@ def upload_blob(self, repository, data, **kwargs):
         :raises ValueError: If the parameter repository or data is None.
         """
         try:
-            _, _, start_upload_response_headers = self._client.container_registry_blob.start_upload(
+            _, _, start_upload_response_headers = self._client.container_registry_blob.start_upload( # type: ignore
```

Comment:
What were the errors here before you had to ignore them?

Reply:
There are usually better ways to fix this than simply ignoring everything.

Reply:
Have changed to just return headers.